src/components/FilterBlock.jsx

```diff
@@ -4,19 +4,19 @@ import { useCharacteristicsContext } from '../hooks/useCharacteristicsContext';
 
 function FilterBlock() {
   const { selectedCharacteristics = [], handleClear } = useCharacteristicsContext();
-  const close = true;
+
   return (
     <div className={style.filter}>
       <div className={style.btn_block}>
         {selectedCharacteristics.map((item, index) => (
-          <ButtonLanguage close={close} key={index}>
+          <ButtonLanguage parentName={FilterBlock} key={index}>
             {item}
           </ButtonLanguage>
         ))}
       </div>
-      <span onClick={() => handleClear()} className={style.clear}>
+      <button onClick={() => handleClear()} className={style.clear}>
```

Comment:
You don't need to create an anonymous function around `handleClear` here since you don't pass any arguments